apps/ejabberd/src/node_push.erl

```diff
@@ -97,25 +98,25 @@ publish_item(Nidx, Publisher, Model, MaxItems, ItemId, ItemPublisher, Payload, P
 
     case is_allowed_to_publish(Model, Affiliation) of
         true ->
-            do_publish_item(Nidx, Publisher, Model, MaxItems, ItemId, ItemPublisher,
+            do_publish_item(ServerHost, Nidx, Publisher, Model, MaxItems, ItemId, ItemPublisher,
                             Payload, PublishOptions);
         false ->
             {error, ?ERR_FORBIDDEN}
     end.
 
-do_publish_item(_Nidx, _Publisher, _Model, _MaxItems, _ItemId, _ItemPublisher,
+do_publish_item(ServerHost, _Nidx, _Publisher, _Model, _MaxItems, _ItemId, _ItemPublisher,
```

Comment:
Can all the vars with `_` prefix be removed? We probably don't need them and without them the function head will be much more readable.